app.py

```diff
@@ -86,6 +133,22 @@ def handleJoin(self, room, message):
             self.sendPm(user.id, self.usernotes.pendingMessages(user.id))
 
     def parseMessage(self, msg, roomName):
+        """Parses the message given by a user and delegates the tasks further
+
+        This is where we handle most of the parsing of the PS protocols. Tasks
+        like user related queries(i.e. commands) are delegated to the Command method.
+        And Showdown Tourneys are handled in their own interface in the plugins module.
+        Likewise for the MessageDatabase inteface.
+
+        Args:
+            room: Room object, room this message was received from.
```

Comment:
First of all, wrong order on these. Second of all, `parseMessage` is passed the room as a string of the room name, which it uses to find the right room object. So this should be `roomName: String, name of the room that the message came from.` or something of the sort.